### PR TITLE
Vitess user survey blog 

### PR DESCRIPTION
**Pull request by kirtanchandak**

### CODE REVIEW

File: content/en/blog/2024-08-27-vitess-user-survey.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+description: "Learn the basics of Change Data Capture (CDC) and how to leverage Vitess's VStream API to build data pipelines."
+---
+
+We recently conducted a survey of how Vitess is being used by the community. This blog post summarizes what we learned.
+
+## Background
+
+The story begins with the Vitess maintainer team wanting to conduct a user survey with community members and engineers using Vitess in production to gain a better understanding of how to improve the project. As an LFX Mentee'24, I worked with the Vitess maintainers to form the questions and create a format for conducting the user survey. We drafted the questions, reviewed them, and developed a final version of the survey. I'm happy to report that we received a number of responses.
+
+## Summary Responses  -
```
Comment (deepthi) [inline content/en/blog/2024-08-27-vitess-user-survey.md]:
```suggestion
## Summary Responses
```

File: content/en/blog/2024-08-27-vitess-user-survey.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+We recently conducted a survey of how Vitess is being used by the community. This blog post summarizes what we learned.
+
+## Background
+
+The story begins with the Vitess maintainer team wanting to conduct a user survey with community members and engineers using Vitess in production to gain a better understanding of how to improve the project. As an LFX Mentee'24, I worked with the Vitess maintainers to form the questions and create a format for conducting the user survey. We drafted the questions, reviewed them, and developed a final version of the survey. I'm happy to report that we received a number of responses.
+
+## Summary Responses  - 
+
+#### 1.How long have you been using Vitess?
```
Comment (deepthi) [inline content/en/blog/2024-08-27-vitess-user-survey.md]:
```suggestion
#### How long have you been using Vitess?
```

File: content/en/blog/2024-08-27-vitess-user-survey.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+First of all, we decided to ask how long our community members have been using Vitess. Interestingly most users have been using Vitess for more than 3 years. 
+
+![1-vitess-long.png](/files/2024-08-27-vitess-user-survey/1-vitess-long.png)
```
Comment (deepthi) [inline content/en/blog/2024-08-27-vitess-user-survey.md]: Can we center this instead of it being left justified?

Comment (kirtanchandak) [reply content/en/blog/2024-08-27-vitess-user-survey.md]: added different charts and it was not looking good in the middle so I kept it left-aligned, you can check the preview, and then make a decision

File: content/en/blog/2024-08-27-vitess-user-survey.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
+---
+author: 'Kirtan Chandak'
+date: 2024-08-27
+slug: '2024-08-27-vitess-user-survey'
+tags: ['Vitess', 'PlanetScale', 'MySQL', 'VReplication', 'CDC', 'Debezium']
+title: 'Vitess User Survey'
+description: "Learn the basics of Change Data Capture (CDC) and how to leverage Vitess's VStream API to build data pipelines."
```
Comment (deepthi) [inline content/en/blog/2024-08-27-vitess-user-survey.md]: Can you rewrite this? Looks like copy/paste.

File: content/en/blog/2024-08-27-vitess-user-survey.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
+---
+author: 'Kirtan Chandak'
+date: 2024-08-27
+slug: '2024-08-27-vitess-user-survey'
+tags: ['Vitess', 'PlanetScale', 'MySQL', 'VReplication', 'CDC', 'Debezium']
```
Comment (deepthi) [inline content/en/blog/2024-08-27-vitess-user-survey.md]:
```suggestion
tags: ['Vitess', 'Sharding', 'MySQL']
```

File: content/en/blog/2024-08-27-vitess-user-survey.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+title: 'Vitess User Survey'
+description: "Learn the basics of Change Data Capture (CDC) and how to leverage Vitess's VStream API to build data pipelines."
+---
+
+We recently conducted a survey of how Vitess is being used by the community. This blog post summarizes what we learned.
+
+# What our users said -
```
Comment (deepthi) [inline content/en/blog/2024-08-27-vitess-user-survey.md]: Let's remove this heading.

File: content/en/blog/2024-08-27-vitess-user-survey.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+
+> **"Vitess solves an existential threat for services which outgrow a single MySQL database."**
+
+> **"Horizontal sharding helps us scale quickly, and the new generation execution plan, Gen4, enables us to support more SQL queries."**
+
+# Background
+
+The story begins with the Vitess maintainer team wanting to conduct a user survey with community members and engineers using Vitess in production to gain a better understanding of how to improve the project. As an LFX Mentee'24, I worked with the Vitess maintainers to form the questions and create a format for conducting the user survey. We drafted the questions, reviewed them, and developed a final version of the survey. I'm happy to report that we received a number of responses.
+
+# Summary Responses  -
```
Comment (deepthi) [inline content/en/blog/2024-08-27-vitess-user-survey.md]:
```suggestion
# Summary Responses
```

File: content/en/blog/2024-08-27-vitess-user-survey.md [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+23% of the users are running version 19.0 and other versions has a distributed split as seen in the graph below.
+<div style="width:500px;">
+<img src="/files/2024-08-27-vitess-user-survey/15-version.png" alt="Vitess Project Booth" style="margin-top:6px"/>
+</div>
+
+# Insights
+
+### We asked our users how Vitess has helped their business. We got some interesting answers from different companies.
+
+1. Scalability: Generally, it has scaled the performance of databases by horizontal sharding.
```
Comment (deepthi) [inline content/en/blog/2024-08-27-vitess-user-survey.md]:
```suggestion
1. Scalability: Many Vitess users have scaled the performance of their databases using horizontal sharding.
```

File: content/en/blog/2024-08-27-vitess-user-survey.md [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+For eg. -
```
Comment (deepthi) [inline content/en/blog/2024-08-27-vitess-user-survey.md]:
```suggestion
For example:
```

File: content/en/blog/2024-08-27-vitess-user-survey.md [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+
+1. Improved Observability: A consistent dashboard for observing the vreplication streams, with controls over schema changes, can give a better overview of operations.
+2. Unified Web UI: A single, easy-to-use web interface for all administrative tasks would ease management and enhance usability.
+3. Circuit Breaking and Adaptive Concurrency Limits: We need features to prevent clients from being blocked by slow shards, and manage concurrency.
+4. More Clear Upgrade Path with LTS Versions: a well-defined upgrade path with LTS versions is important for stability and fewer bugs.
+5. Auto Sharding Improvements: More advanced auto-sharding capabilities would be nice, such as in TiDB.
+6. Vitess Keyspace History: Add a feature to track and review changes over time to a keyspace—such as reparents, backups, or scaling events for historical analysis and debugging.
+
+# Conclusion
+We appreciate all of you for your time to fill up the survey and share the inputs from your side. Your feedback has provided us with crucial findings that will guide our future efforts.
```
Comment (deepthi) [inline content/en/blog/2024-08-27-vitess-user-survey.md]:
```suggestion
The maintainer team appreciates your responses to the survey. Your feedback will be incorporated into the project roadmap.
```

Comment (kirtanchandak) [reply content/en/blog/2024-08-27-vitess-user-survey.md]: made the changes.

**Review by deepthi — APPROVED** [review]

**Comment by deepthi** [comment]
To be merged on Aug 27.

**Pull request merged**